lib/internal/modules/helpers.js

```diff
@@ -320,6 +320,37 @@ function normalizeReferrerURL(referrerName) {
 }
 
 
```

Comment:
It would be better to serialize the set of strings from C++ into JS instead of duplicating them in JS, otherwise they can get out of sync. Also, there is already something like this in module_job.js for detecting the declaration errors, that can just be merged into the set here.